src/Uniphar.Dotnet.Cli/Commands/GetTokenCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Uniphar.Dotnet.Cli.Token;
```

Comment:
Given we've got such a small codebase here, I think it's worth keeping everything in the same namespace (`Uniphar.Dotnet.Cli`) - the guidance is generally that namespaces should reflect large areas of functionality/subsystems, not simply mimic the folder structure. It also means we don't have to sprinkle `using`'s around to import single classes.

Same applies to `CommandAppExtensions`, and then we can remove the `using`s in `Program.cs` 👍

Reply:
In the past I was organizing the folder structure based on domain separation or responsibilities, but after reading your comment, I realize that aligning namespaces with these doesn't provide much real benefit. Keeping everything in the same namespace makes it cleaner and avoids unnecessary using statements. Especially when the codebase is not huge